openfisca_france_local/parameters/regions/hauts_de_france/crous/aide_100_repas_gratuits.yml

```diff
@@ -0,0 +1,20 @@
+age:
+  description: Limite d'âge de l'aide du crous et de la région Hauts-de-France de 100 repas gratuits aux étudiants boursiers
+  maximum:
+    2023-09-01:
+      value: 35
+
+echelon_boursier:
+  description: Echelons autorisés pour l'aide du crous et de la région Hauts-de-France de 100 repas gratuits aux étudiants boursiers 
+  minimum:
+    2023-09-01:
+      value: 3
+  maximum:
+    2023-09-01:
+      value: 7
+
+metadata:
+  reference:
+    2023-09-01:
+      href: https://www.ij-hdf.fr/actualite/747/100-repas-gratuits-de-nouveau-attribues-aux-etudiants-boursiers-echelon-3-a-7#:~:text=Le%20Conseil%20R%C3%A9gional%20Hauts-de,en%20restos
+      
```

Comment:
En fait quand je checkout le code je vois bien un soucis encore, la dernière ligne contient 6 espaces sans saut de ligne à la fin, tu utilise quel éditeur de texte ? pour pallier à ce soucis (c'est une convention [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)) perso j'ai des options dans vscode : 
- Files: Trim Trailing Whitespace
- Files: Insert Final Newline
- Files: Trim Final Newlines
d'activées, (et du coup je me pose plus la question)

Reply:
J'ai vscode insiders. Merci pour le tips, j'ai activé ces options et ça devrait être bon

Reply:
Mais manque plus que ce fichier 😅

Reply:
Oups, lui il est passé à la trappe ^^